ci/common.sh

```diff
@@ -20,6 +20,7 @@ set -ex
 export_or_prefix() {
     export OPENRESTY_PREFIX="/usr/local/openresty-debug"
     export APISIX_MAIN="https://raw.githubusercontent.com/apache/incubator-apisix/master/rockspec/apisix-master-0.rockspec"
+    export APISIX_LDAP_DEPRECATION="https://raw.githubusercontent.com/Zhenye-Na/apisix/remove-openldap/rockspec/apisix-master-0.rockspec"
```

Comment:
Please remove the personal link and use the local file apisix-master-0.rockspec.

Reply:
If I remove this reference to use the apache/apisix master branch, `lualdap` will be included during the dependencies installation. This will cause the issue happened before, which is `lualdap.h` is missing the configuration

Reply:
Though I am not too sure regarding the `ldap-auth` testing failures as I was not touching any files that are referencing the dependencies of `ldap-auth`. Maybe I am missing something ?